modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/paths/TestrayBuild.path

```diff
@@ -0,0 +1,36 @@
+<html>
+<head>
+<title>TestrayBuild</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">TestrayBuild</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>ROUTINE_ID</td>
+	<td>//select[contains(@id, 'routineId')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>PRODUCT_VERSION</td>
+	<td>//select[contains(@id, 'productVersionId')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>GIT_HASH_FIELD</td>
+	<td>//input[contains(@id, 'gitHash')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>BUILD_DETAILS</td>
+	<td>//div[contains(@class, 'panel-collapse')]//table[contains(@class, 'qa w-100')]</td>
```

Comment:
Is there a more descriptive way to find this element? The class "qa w-100" can easily change and the test will need a test fix?